apps/ledger-live-mobile/src/components/AccountCard.tsx

```diff
@@ -39,9 +41,11 @@ const AccountCard = ({
   ...props
```

Comment:
You can remove parentAccount from the props since you don't use it anymore

Reply:
update it here too apps/ledger-live-mobile/src/newArch/features/Web3Hub/screens/Web3HubApp/components/Web3Player/SelectAccountModal/AccountsList/index.tsx

Reply:
It's used direclty inside component :) So no need to update other file

btw i'll update logic because maybe in some cases parentAccount will not be undefined